src/server/scripts/Spells/spell_druid.cpp

```diff
@@ -197,6 +204,92 @@ class spell_dru_enrage : public SpellScriptLoader
         }
 };
 
+// 37336 - Druid Forms Trinket
+class spell_dru_forms_trinket : public SpellScriptLoader
+{
+public:
+    spell_dru_forms_trinket() : SpellScriptLoader("spell_dru_forms_trinket") { }
+
+    class spell_dru_forms_trinket_AuraScript : public AuraScript
+    {
+        PrepareAuraScript(spell_dru_forms_trinket_AuraScript);
+
+        bool Validate(SpellInfo const* /*spellInfo*/) override
+        {
+            if (!sSpellMgr->GetSpellInfo(SPELL_DRUID_FORMS_TRINKET_BEAR)
+                || !sSpellMgr->GetSpellInfo(SPELL_DRUID_FORMS_TRINKET_CAT)
+                || !sSpellMgr->GetSpellInfo(SPELL_DRUID_FORMS_TRINKET_MOONKIN)
+                || !sSpellMgr->GetSpellInfo(SPELL_DRUID_FORMS_TRINKET_NONE)
+                || !sSpellMgr->GetSpellInfo(SPELL_DRUID_FORMS_TRINKET_TREE))
+                return false;
+            return true;
+        }
+
+        bool CheckProc(ProcEventInfo& eventInfo)
+        {
+            if (Unit* target = eventInfo.GetActor())
```

Comment:
I dont think it is possible... it must always return something now that I think about it... @Shauren you think so?
In case I can just remove those checks